### ISSUE TITLE
missing rubric.json

### ISSUE THREAD

**Issue by pinventado** [ILXL/rubricscore#3]
For the ILXL repo, most problems only have unit tests and it's hard to manually create rubric.json files for each one (there are about 170 problems right now). 

Is it possible for us to provide a default grade for all unit tests if a rubric.json file is not present? For example, by default each test found in the .xml file gets 1 point. The total is just the sum of all the points.

**Comment by kevinwortman** [comment]
Yes it's possible. We should probably change the commandline UI so that you can either provide a
-- rubric <JSON>
or
-- unweighted
(or similar word)
flag to specify how points are assigned.

**Comment by pinventado** [comment]
Cool! @NamanhTran we should look into this feature next :)
